src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/tests/test_iot_commands.py

```diff
@@ -25,13 +26,18 @@ def body(self):
                      JMESPathCheck('name', hub),
                      JMESPathCheck('sku.name', 'S1')
                  ])
+        connection_string = 'HostName=iot-hub-for-testing.azure-devices.net;SharedAccessKeyName=iothubowner;SharedAccessKey=P80tG5ljXUHtGYyc+oP3GCAazRCq8d0VUtJ4RGrSru4='
+        self.cmd('iot hub show-connection-string -n {0} -g {1}'.format(hub, rg),
+                 checks=[
+                     JMESPathCheck('connectionString', connection_string)
+                 ])
 
 
 class IoTDeviceCreateTest(VCRTestBase):
```

Comment:
>VCRTestBase [](start = 26, length = 11)

Instead of using the VCRTestBase, please use the ResourceGroupVCRTestBase. This test requires someone to have created the specific resource group in advance. If someone needs to re-record a test, it is best practice if those setup and clean up tasks are handled automatically. Otherwise tests fail because the RG doesn't exist or the RG lingers because the test didn't clean it up.